doc/source/serve/model_composition.md

```diff
@@ -212,65 +212,52 @@ $ python arithmetic_client.py
 9
 ```
 
-(deployment-graph-call-graph-testing)=
-### Testing the Call Graph with the Python API
-
-All `MethodNodes` and `FunctionNodes` have an `execute` method. You can use this method to test your graph in Python, without using HTTP requests. 
+(deployment-graph-drivers-http-adapters)=
+### Drivers and HTTP Adapters
 
-To test your graph,
+Ray Serve provides the `DAGDriver`, which routes HTTP requests through your call graph. As mentioned in [the call graph section](deployment-graph-call-graph), the `DAGDriver` takes in a `DeploymentNode` and it produces a `ClassNode` that you can run.
 
-1. Call `execute` on the `MethodNode` or `FunctionNode` that you would pass into the `DAGDriver`.
-2. Pass in the input to the graph as the argument. **This argument becomes the input represented by `InputNode`**. Make sure to refactor your call graph accordingly, since it takes in this input directly, instead of an HTTP request.
-3. `execute` returns a reference to the result, so the graph can execute asynchronously. Call `ray.get` on this reference to get the final result.
+The `DAGDriver` also has an optional keyword argument: `http_adapter`. [HTTP adapters](serve-http-adapters) are functions that get run on the HTTP request before it's passed into the graph. Ray Serve provides a handful of these adapters, so you can rely on them to conveniently handle the HTTP parsing while focusing your attention on the graph itself.
 
-As an example, we can rewrite the [arithmetic call graph example](deployment-graph-arithmetic-graph) from above to use `execute`:
+For instance, we can use the Ray Serve-provided `json_request` adapter to simplify our [arithmetic call graph](deployment-graph-arithmetic-graph) by eliminating the `unpack_request` function. You can replace lines 29 through 38 with this graph:
```

Comment:
There is inconsistent use of second person singular "you" and second person plural "we." 

We might want to be consistent to only use.

Reply:
Good catch– I replaced usages of "we" and "our" with "you" and "your."